test/DataFileUtil_server_test.py

```diff
@@ -215,6 +215,45 @@ def test_unpack(self):
         self.assertEqual(ret1['file_path'],
                          str(os.path.join(unpack_dir, 'file1.txt')))
 
+    def test_unpack_large_zip(self):
+        txt_filename = 'large_file.txt'
+        zip_filename = 'large_file.txt.zip'
+        tmp_dir = os.path.join(self.cfg['scratch'], 'unpacklargeziptest')
+        if not os.path.exists(tmp_dir):
+                os.makedirs(tmp_dir)
+        zip_file_path = os.path.join(tmp_dir, zip_filename)
+        txt_file_path = os.path.join(tmp_dir, txt_filename)
+
+        size_2GB = 2 * 1024 * 1024 * 1024
```

Comment:
👍 
It was quite slow previously. So I want to make the test file just barely larger than 2GB. But I just find a way to quickly generate a large legit zipfile. Tests runs within acceptable time now.